clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTaskTest.java

```diff
@@ -2071,37 +2070,6 @@ public void testKillConsumption(AAConfig aaConfig) throws Exception {
     }
   }
 
-  @Test(dataProvider = "aaConfigProvider")
-  public void testKillActionPriority(AAConfig aaConfig) throws Exception {
```

Comment:
Is it possible to set "SERVER_PROCESS_CONSUMER_ACTION_WITHOUT_ENQUEUE_ENABLE" as false and keep this test?